docs/configuration/radio/device.mdx

```diff
@@ -28,7 +28,7 @@ Acceptable values:
 |     `SENSOR`     |                                                                                                        Sensor - For use with devices intended to primarily collect sensor readings. Telemetry packets sent from this device will be higher priority, broadcasting every five minutes. When used in conjunction with power.is_power_saving = true, nodes will wake up, send environment telemetry, and then sleep for telemetry.environment_update_interval seconds.                                                                                                        |
 |      `TAK`       |                                                                                                                                                                                        TAK - Used for nodes dedicated for connection to an ATAK EUD.  Turns off many of the routine broadcasts to favor CoT packet stream from the Meshtastic ATAK plugin -> IMeshService -> Node.                                                                                                                                                                                         |
 | `CLIENT_HIDDEN`  |                                                                                                                            Client Hidden - Used for nodes that "only speak when spoken to." Turns off all of the routine broadcasts but allows for ad-hoc communication. Still rebroadcasts, but with local only rebroadcast mode (known meshes only). Can be used for clandestine operation or to dramatically reduce airtime / power consumption                                                                                                                             |
-| `LOST_AND_FOUND` |                                                                                                                                                                                                  Lost and Found - Used to automatically send a text message to the mesh with the current position of the device on a frequent interval: "I'm lost! Position: lat / long"                                                                                                                                                                                                   |
+| `LOST_AND_FOUND` |                                                                                                                                                                                                  Lost and Found - Used to automatically send a text message on the primary channel for the device with the current position of the device on a frequent interval: "I'm lost! Position: lat / long"                                                                                                                                                                                                   |
```

Comment:
I was about to merge this but realized you use 'device' twice.  Would you consider:

"Used to automatically send a text message with current position at frequent intervals to the primary channel for the device: "I'm lost! Position: lat / long""

Reply:
Yes, I think I've changed the pull request to this language. Thanks for making it clearer.